spec/unit/graph/simple_graph_spec.rb

```diff
@@ -528,31 +528,31 @@ def empty_graph(graph)
     end
 
     def one_vertex_graph(graph)
-      graph.add_vertex(:a)
```

Comment:
Are there any usages of SimpleGraph where we need to preserve symbols? I don't think the Catalog relies on that as the vertices are either `Puppet::Resource` objects in the resource (server) catalog, or `Puppet::Type::<name>` instances in the RAL (agent) catalog.